lib/registries/addon/overview/template.hbs

```diff
@@ -138,6 +138,49 @@
                     <leftNav.link data-analytics-name='Comments' @route='registries.overview.comments'
                         @models={{array this.registration.id}} @icon='comments'
                         @label={{t 'registries.overview.comments.title'}} @count={{this.commentsCount}} />
+
+                    <div data-test-resources-wrapper local-class='ResourcesWrapper'>
+                        <div local-class='ResourcesWrapper_title'>
+                            <h3>{{t 'registries.overview.resources.left_nav_title'}}</h3>
+                            <FaIcon local-class='HelpIcon' @icon='question-circle' />
+                            <BsTooltip
+                                @placement='right'
+                                @triggerEvents='hover'
+                            >
+                                {{t 'registries.overview.resources.help_icon'}}
+                            </BsTooltip>
+                        </div>
+                        <OpenResource
+                            data-test-data-resource
+                            @hasResource={{this.registration.hasData}}
```

Comment:
See comment above about making `hasResource` a computed property for the component.